CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- Remove the code introduced by version `3.105.0`.
+
 ## [3.105.0] - 2021-07-15
```

Comment:
Can you also edit here to have this set to YANKED? e.g.: https://github.com/vtex-apps/search-result/blob/4d12b38877dd95531c0ea02110239531c656ecb3/CHANGELOG.md#3841---2020-11-11-yanked

```suggestion
## [3.105.0] - 2021-07-15 [YANKED]
```

Reply:
I added but I had to removed it in my last commit https://github.com/vtex-apps/search-result/pull/530/commits/4d12b38877dd95531c0ea02110239531c656ecb3.

The Danger action was complaining that I shouldn't do this because it will be done automatically